arviz/tests/test_plots.py

```diff
@@ -7,7 +7,7 @@
 import pytest
 import pymc3 as pm
 
-
+from ..data import load_arviz_data

```

Comment:
Being nitpicky here, can this import be combined with the import from `..data` below?